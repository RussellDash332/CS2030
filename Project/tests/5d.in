1 2 0 3 20 1.0 1.0 0.1 0.5 0.9