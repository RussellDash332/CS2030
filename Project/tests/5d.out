0.000 1(greedy) arrives
0.000 1(greedy) serves by server 1
0.313 1(greedy) done serving by server 1
0.314 2(greedy) arrives
0.314 2(greedy) serves by server 1
0.417 2(greedy) done serving by server 1
1.205 3(greedy) arrives
1.205 3(greedy) serves by server 2
1.904 3(greedy) done serving by server 2
2.776 4(greedy) arrives
2.776 4(greedy) serves by server 2
2.791 4(greedy) done serving by server 2
3.877 5(greedy) arrives
3.877 5(greedy) serves by server 1
3.910 6(greedy) arrives
3.910 6(greedy) serves by server 2
3.922 6(greedy) done serving by server 2
4.031 5(greedy) done serving by server 1
9.006 7(greedy) arrives
9.006 7(greedy) serves by server 1
9.043 8(greedy) arrives
9.043 8(greedy) serves by server 2
9.105 9(greedy) arrives
9.105 9(greedy) waits at server 1
9.160 10 arrives
9.160 10 waits at server 1
9.225 11(greedy) arrives
9.225 11(greedy) waits at server 2
10.148 12(greedy) arrives
10.148 12(greedy) waits at server 2
10.484 7(greedy) done serving by server 1
10.484 9(greedy) serves by server 1
10.781 9(greedy) done serving by server 1
11.205 13(greedy) arrives
11.205 13(greedy) waits at server 1
11.636 8(greedy) done serving by server 2
11.636 11(greedy) serves by server 2
11.688 11(greedy) done serving by server 2
11.688 12(greedy) serves by server 2
12.429 14(greedy) arrives
12.429 14(greedy) waits at server 2
13.109 15(greedy) arrives
13.109 15(greedy) waits at server 2
14.644 10 serves by server 1
15.013 10 done serving by server 1
15.178 12(greedy) done serving by server 2
15.178 14(greedy) serves by server 2
15.264 16 arrives
15.264 16 waits at server 1
15.338 14(greedy) done serving by server 2
15.338 15(greedy) serves by server 2
15.524 17(greedy) arrives
15.524 17(greedy) waits at server 2
15.940 18(greedy) arrives
15.940 18(greedy) waits at server 2
17.793 19(greedy) arrives
17.793 19(greedy) waits at server 1
18.207 15(greedy) done serving by server 2
18.207 17(greedy) serves by server 2
18.765 20(greedy) arrives
18.765 20(greedy) waits at server 2
19.103 17(greedy) done serving by server 2
19.103 18(greedy) serves by server 2
20.146 18(greedy) done serving by server 2
40.474 13(greedy) serves by server 1
40.480 13(greedy) done serving by server 1
40.480 16 serves by server 1
41.056 16 done serving by server 1
44.634 19(greedy) serves by server 1
45.376 19(greedy) done serving by server 1
57.110 20(greedy) serves by server 2
60.117 20(greedy) done serving by server 2
[7.065 20 0]
